release/README-development-process.html

```diff
@@ -54,24 +58,11 @@ <h2 align="center">Checker Framework Development Process</h2>
                    do not have performance regressions compared to the previous release (this list may be narrowed down to fewer
                    projects in the future). If a performance regression was introduced, work with the feature owner to understand
                    whether bug fixes can fix the performance regression and to decide on the appropriate action.</li>
+                   <li>Sign in to <a href="https://travis-ci.org/">Travis</a> and ensure that all the projects in the left-hand column are
```

Comment:
Not all the projects; for instance, Randoop's tests failing doesn't block a Checker Framework release.  I think it's better to say all the ones in the typetools organization (are there others that are important?)